packages/react-native/lib/truvera-biometric-plugin.ts

```diff
@@ -30,6 +30,12 @@ export const getIssuanceDate = () => {
   return dateObject.toISOString();
 };
 
+function getBiometricCreatedDate() {
```

Comment:
this could probably be more generic if you pass the datetime in and convert it

```
function convertDateTimeToDate(dt) {
  return dt.split('T')[0];
}
```